internal/config/storageclass/storage-class.go

```diff
@@ -90,6 +106,8 @@ type Config struct {
 	Standard    StorageClass `json:"standard"`
 	RRS         StorageClass `json:"rrs"`
 	Optimize    string       `json:"optimize"`
+	inlineBlock int64
```

Comment:
So this isn't stored, correct?

Reply:
This config has nothing to do with storing.